### ISSUE TITLE
Add issues for what is going on and what probably will go on in the project

### ISSUE THREAD

**Issue by rhagman** [rhagman/sustainable-green-plants#24]
To be more inclusive in the project and help with collaboration, it is nice to have an open place to share what the current plan is for the project. Creating issues and structure them in different projects here in github is probably a good way to do it.

**Comment by WebbPeter** [comment]
I agree! It's definitely valuable to cross pollinate (hah) platforms though! Definitely make sure that people on your slack, as well as your general blog know that you want them to participate.
